### PR TITLE
allow to specify aliases for the vagrant host

### PR DESCRIPTION
**Pull request by plandolt**

### CODE REVIEW

**Comment by plandolt** [comment]
ready for review

File: Vagrantfile [hunk 1]
```diff
@@ -112,11 +112,21 @@ Vagrant.configure(2) do |config|
...
 
-    if not setupConfig['subhosts'].empty?
+    if not setupConfig['subhosts'].empty? or not setupConfig['aliases'].empty?
         aliases = []
-        setupConfig['subhosts'].each do |subhost|
-            aliases.push(subhost['subhostname'] + '.' + setupConfig['hostname'])
+
+        if not setupConfig['subhosts'].empty?
+            setupConfig['subhosts'].each do |subhost|
+                aliases.push(subhost['subhostname'] + '.' + setupConfig['hostname'])
```
Comment (dominikzogg) [inline Vagrantfile]: no aliases for subhosts?

**Review by dominikzogg — COMMENTED** [review]
https://github.com/vagrant-php/ubuntu/pull/43/files#diff-c87e890cac26440a8d79a0a0b835a79dR66

alias missing

**Review by dominikzogg — CHANGES_REQUESTED** [review]
https://github.com/vagrant-php/ubuntu/pull/43/files#diff-c87e890cac26440a8d79a0a0b835a79dR66

alias missing

File: Vagrantfile (hunk 1 above)
Comment (plandolt) [reply Vagrantfile]: I thought of it as you need either subhosts or aliases but not both. How would the final setup look like if you want aliases and subhosts?

Comment (dominikzogg) [reply Vagrantfile]: host
alias
subhost.host
subhost.alias

**Comment by plandolt** [comment]
now you can define global aliases and aliases for subhosts.

**Comment by plandolt** [comment]
Ping @dominikzogg ?

**Review by dominikzogg — APPROVED** [review]

**Comment by dominikzogg** [comment]
@scuben thanks for the contribution

**Pull request merged**